muikku-core-plugins/src/main/java/fi/otavanopisto/muikku/plugins/transcriptofrecords/TranscriptOfRecordsController.java

```diff
@@ -41,7 +45,39 @@
   @Inject
   private SearchProvider searchProvider;
   
+  @Inject
+  private Logger log;
+  
   private static final Pattern UPPER_SECONDARY_SCHOOL_SUBJECT_PATTERN = Pattern.compile("^[A-ZÅÄÖ0-9]+$");
+  
+  @PostConstruct
+  public void init() {
+    String commaSeparatedSubjectsOrder = pluginSettingsController.getPluginSetting("transcriptofrecords", "subjectsOrder");
+    if (commaSeparatedSubjectsOrder == null) {
```

Comment:
StringUtils.isBlank/isEmpty? Or is it by design that there's no warning if the pluginSetting is empty?